working/augmentation-libraries/feature-specification.md

```diff
@@ -145,6 +145,17 @@ It is a compile-time error if:
 *   A library augmentation contains a normal `library` directive. They are not
     self-contained libraries, only pieces of the augmented library.
 
+*   An `import augment` directive has a `<uri>` that denotes an entity which
+    is not an augmentation library. *For example, it can not be a library.*
```

Comment:
We recently changed the terminology from "augmentation library" to "library augmentation" to make it clearer that augmentations are not libraries, so change that here and throughout.

Reply:
That's great, 'library augmentation' seems to nudge the brain into exactly the right perspective! Done.